docker-compose.yml

```diff
@@ -0,0 +1,217 @@
+version: "3.7"
+
+services:
+
+  # the ui-only container will connect to the second instance
+  joinmarket-webui-ui-only:
+    build:
+      context: ./ui-only
+      dockerfile: Dockerfile
+      args: 
+        JM_UI_REPO: ${JM_UI_REPO?err}
+        JM_UI_REPO_BRANCH: ${JM_UI_REPO_BRANCH?err}
+        JM_UI_REPO_REF: ${JM_UI_REGRESSION_TEST_REPO_REF:-v0.0.2}
+    restart: on-failure
+    stop_grace_period: 1m
+    environment:
+      JMWEBUI_JM_WALLETD_HOST: joinmarket-webui-standalone2
+      JMWEBUI_JM_WALLETD_PORT: 28183
+    ports:
+      - "8081:80"
+
+  joinmarket-webui-standalone:
+    build:
+      context: ./standalone
+      dockerfile: Dockerfile
+      args: 
+        JM_UI_REPO: ${JM_UI_REPO?err}
+        JM_UI_REPO_BRANCH: ${JM_UI_REPO_BRANCH?err}
+        JM_UI_REPO_REF: ${JM_UI_REPO_REF?err}
+        JM_SERVER_REPO: ${JM_SERVER_REPO?err}
+        JM_SERVER_REPO_BRANCH: ${JM_SERVER_REPO_BRANCH?err}
+        JM_SERVER_REPO_REF: ${JM_SERVER_REPO_REF?err}
+    restart: unless-stopped
+    environment:
+      ENSURE_WALLET: 1
+      READY_FILE: /root/.nbxplorer/btc_fully_synched
+      jm_gaplimit: 2000
+      jm_max_cj_fee_abs: 42000000 # 0.42 btc
+      jm_max_cj_fee_rel: 0.1337 # 13.37%
```

Comment:
1. For `jm_max_cj_fee_abs` having a arbitrary high value seems ok, for `jm_max_cj_fee_rel` a reasonable value would be way lower, but …
2. As this will vary (and potentially trend downwards with time), it'd make sense to have it as an orderbook-adjusted value in the UI. We discussed this previously and joinmarket-webui/joinmarket-webui#14 is the prerequisite for that feature.

Reply:
Just for the record: This is only in the docker-compose setup, so it has nothing to do with what will end up in the containers at runtime.

E.g. I have set other (hopefully more sane) values for umbrel: https://github.com/getumbrel/umbrel/pull/1216/files#diff-ce0b76634411353aa10b2d5e9dddcbe0c70e646f40a6218b25a4721747679136R32-R33

This is by no means something I investigated deeply - the default values should be discussed and settled on properly. But this image is currently not responsible for it. It would be another story if we provide a default config in the image (which we currently don't do).

Reply:
> Just for the record: This is only in the docker-compose setup, so it has nothing to do with what will end up in the containers at runtime.

At the risk of this being a stupid question: When you say "only in the docker-compose setup" you mean the regtest dev setup?

Reply:
> When you say "only in the docker-compose setup" you mean the regtest dev setup?

I mean the docker-compose.yml file the referenced lines are from (in this commit).
This is a copy of the docker-compose "regtest dev env" we have in the "joinmarket-webui" repo for UI development, but this one uses the now newly built images. The images itself are network agnostic (at least they should be, as everything can be adapted with environment variables). Hope this answers your question.